sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -332,6 +332,43 @@ public void disablePersisted() {
         verify(channel, never()).addListener(any(Channel.Listener.class));
     }
 
+    @Test
+    public void notSendingLogsOnPause() {
+
+        /* Before start it does not work to change state, it's disabled. */
+        Analytics analytics = Analytics.getInstance();
+
+        /* Start. */
+        Channel channel = mock(Channel.class);
+        analytics.onStarting(mAppCenterHandler);
+        analytics.onStarted(mock(Context.class), channel, "", null, true);
+        verify(channel).addGroup(eq(analytics.getGroupName()), anyInt(), anyLong(), anyInt(), isNull(Ingestion.class), any(Channel.GroupListener.class));
+        verify(channel).addListener(isA(SessionTracker.class));
+        verify(channel).addListener(isA(AnalyticsValidator.class));
+
+        /* Pause Analytics. */
+        Analytics.pause().get();
+
+        /* Check if Analytics group is paused. */
+        verify(channel).pauseGroup(eq(analytics.getGroupName()));
+
+        /* Now try to use all methods. Should work. */
```

Comment:
This comment and the one on line 366 should be updated.  We don't need to test all methods here; it would be redundant.